addons/l10n_my_edi_extended/models/account_edi_xml_ubl_my.py

```diff
@@ -0,0 +1,74 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, api
+
+
+class AccountEdiXmlUBLMyInvoisMY(models.AbstractModel):
+    _inherit = "account.edi.xml.ubl_myinvois_my"
+
+    def _export_invoice_vals(self, invoice):
+        # EXTENDS 'l10n_my_edy'
```

Comment:
I find these comments useless for multiple reasons, but it's even worse if there is a typo
```suggestion
        # EXTENDS 'l10n_my_edi'
```